IPython/core/completer.py

```diff
@@ -168,6 +168,12 @@
 
 _deprecation_readline_sentinel = object()
 
+names = []
+for c in range(0,0x10FFFF + 1):
+    try:
+        names.append(unicodedata.name(chr(c)))
```

Comment:
You used `char` here instead of `chr`. I fixed it.

Reply:
Thanks!